### DISCUSSION TITLE
s/django-celery-beat-queues-priority

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#33] [Announcements]
# s/django-celery-beat-queues-priority

Celery多队列解决生产环境下的任务优先级问题

https://blog.ops-coffee.cn/s/django-celery-beat-queues-priority

**Comment by immelon0097** [comment]
请问想要学习运维系统开发可以从哪里入手？

**Reply by ops-coffee** [reply]
每个人的情况不一样，这个很难回答啊，博客主页右上角有个提问，可以加我微信，详细聊聊
